app/controllers/users_controller.rb

```diff
@@ -247,4 +248,12 @@ def set_email_preferences
       @email_prefs[pref.email_type] = false
     end
   end
+
+  def change_privacy_params(key, value)
```

Comment:
One could also use this to update other user params, so I'd use another name for the method.